entities/src/main/java/com/flipkart/varadhi/entities/auth/RoleBindingNode.java

```diff
@@ -15,8 +15,8 @@
 
 @Getter
 @EqualsAndHashCode(callSuper = true)
-@ValidateVaradhiResource(message = "Invalid Role Binding name. Check naming constraints.")
-public class RoleBindingNode extends VaradhiResource {
+@ValidateResource(message = "Invalid Role Binding name. Check naming constraints.")
+public class RoleBindingNode extends MetaStoreEntity {
```

Comment:
Is it validatable or not as it has all constraints in-place ?
If it is, then mostly default values of length wouldn't suffice as resourceId is concatenation of multiple components.

Reply:
rolebinding node only extends metastore as it is not user facing on the request side
rather I have made IAMPolicyUpdateRequest implement Validatable